aggregate/housing_production/hpd_housing_ny_affordable_housing.py

```diff
@@ -20,28 +22,28 @@
 
 def load_housing_ny():
     """load the HPD Housing NY Units by Building dataset"""
-    df = pd.read_csv(
-        ".library/hpd_hny_units_by_building.csv",
-        usecols=[
-            "project_id",
-            "project_name",
-            "project_start_date",
-            "project_completion_date",
-            "number",
-            "street",
-            "borough",
-            "latitude_(internal)",
-            "longitude_(internal)",
-            "building_completion_date",
-            "reporting_construction_type",
-            "extremely_low_income_units",
-            "very_low_income_units",
-            "low_income_units",
-            "moderate_income_units",
-            "middle_income_units",
-            "other_income_units",
-        ],
-    )
+    df = read_from_S3("hpd_hny_units_by_building",
+                      cols=[
+                          "project_id",
+                          "project_name",
+                          "project_start_date",
+                          "project_completion_date",
+                          "number",
+                          "street",
+                          "borough",
+                          "latitude_(internal)",
+                          "longitude_(internal)",
+                          "building_completion_date",
+                          "reporting_construction_type",
+                          "extremely_low_income_units",
+                          "very_low_income_units",
+                          "low_income_units",
+                          "moderate_income_units",
+                          "middle_income_units",
+                          "other_income_units",
+                      ],
+                      )
+    dump_metadata()
```

Comment:
This implementation keep overwriting the metadata folder every time it's called. This will work but I feel like there's a cleaner, more intuitive way? Is there somewhere downstream we could call this just once?

Reply:
I think by metadata folder you meant the metadata file. I thought the process is keep dumping new information into the same metadata.yml file. We might risk having duplicate but shouldn't overwrite the same file.

Reply:
I do mean metadata file. Did I misunderstand the process? Does this code add to the existing `metdata.yml`? If so I think that's a good approach

Reply:
After our conversation I see that the function is named and implemented correctly